### PR TITLE
t-RdfToFiles configuration class simplification

### PR DESCRIPTION
**Pull request by skodape**
Simplification of configuration class. Added RDF configuration.

The original configuration class - V1 contains unused fields and was not suitable for easy RDF configuration.

Recreation of https://github.com/UnifiedViews/Plugins/pull/216 from current develop.

### CODE REVIEW

File: t-rdfToFiles/README.md [hunk 1]
```diff
@@ -40,6 +40,7 @@
 
 |Version            |Release notes                                   |
 |-------------------|------------------------------------------------|
+|2.2.0              | Added support for RDF configuration. Migration to new configuration version as the old contains unused field. |
```
Comment (skrchnavy) [inline t-rdfToFiles/README.md]: @skodape please merge description with 2.1.0 (such version was not released)

File: t-rdfToFiles/pom.xml [hunk 2]
```diff
@@ -12,7 +12,7 @@
 	<artifactId>uv-t-rdfToFiles</artifactId>
 	<name>T-RdfToFiles</name>
 	<description>Transform RDF graphs into files.</description>
-	<version>2.1.0-SNAPSHOT</version>
+	<version>2.2.0-SNAPSHOT</version>
```
Comment (skrchnavy) [inline t-rdfToFiles/pom.xml]: shall stay 2.1.0-SNAPSHOT

File: t-rdfToFiles/README.md (hunk 1 above)
Comment (mkl-eea-sk) [inline t-rdfToFiles/README.md]: I agree with @skrchnavy , lets do not raise version once it is in SNAPSHOT state. Only when we move from stable to -SNAPSHOT we have to raise version (after release and new changes in develop)

**Comment by tomas-knap** [comment]
Since 2.1 of the DPU was not yet released, we can put all the news to 2.1, no need to 2.2 for the DPU. Otherwise Ok.

**Comment by skrchnavy** [comment]
latest released version is 2.0.1, see http://maven.eea.sk/artifactory/public/eu/unifiedviews/plugins/uv-t-rdfToFiles/ for more details.

**Pull request merged**